wallet/test/run-wallet.c

```diff
@@ -42,16 +42,6 @@ void bitcoind_getutxout_(struct bitcoind *bitcoind UNNEEDED,
 				    void *arg) UNNEEDED,
 			 void *arg UNNEEDED)
 { fprintf(stderr, "bitcoind_getutxout_ called!\n"); abort(); }
-/* Generated stub for blinding_hash_e_and_ss */
-void blinding_hash_e_and_ss(const struct pubkey *e UNNEEDED,
```

Comment:
I don't think you want to remove these, they're needed for your exp_features stuff